src/styles/base/_04-base.scss

```diff
@@ -1,9 +1,11 @@
-html {
-    box-sizing: border-box;
-}
+.nypl-ds {
+    html {
```

Comment:
I think you want `&html { ` or rather `html.nypl-ds` unless if you are thinking of putting a wrapper div around `html` which doesn't sounds right.

Reply:
🤦🏻‍♀️ thank you Edwin